### PR TITLE
Hide logged out message after initial render 

### LINKED ISSUE #598: "Logged out successfully" alert should go away after /login re-renders
# Bug Report
## What is the current behavior?
If you logout, and then try to login again (but fail), the whole page re-renders, but the `loggedOut` query param still exists in the URL and the Alert still renders.

## What is the expected behavior?
After logging out and attempting to log back in, but getting an error (or refreshing the page), the Alert should go away and the URL should read: `https://www.operationcode.org/login`

## What steps did you take to get this behavior?
I logged out from a valid account, and then tried to login using Google but received an error.

### Operating System
MacOS latest

### Browser
Chrome latest

### Is This A Third-Party Issue?
No

### Screenshots
Screen Shot 2019-06-12 at 1 36 06 AM

### PR DESCRIPTION
**Pull request by AllenAnthes**
# Description of changes

Hides logged out alert after initial render by setting the alert message on initial mount, and removing it during any login action.

# Issue Resolved

Fixes #598

### CODE REVIEW

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit 75b9e31fe274739600dd94da355e9cf2547e57ce

https://deploy-preview-613--operation-code-storybook.netlify.com

**Comment by kylemh** [comment]
Every JS file in the pages folder gets bundled into now - even test files.

I’d recommend simply adding to the Cypress tests, since we don’t add the pages folder to test coverage. If you’re set on Jest, you’ll want to make a folder outside of pages for tests of those files.

File: cypress/integration/login.spec.js [hunk 1]
```diff
@@ -90,3 +90,39 @@ describe('login', () => {
     cy.getCookies().should('have.length', 0);
   });
 });
+
+describe('login?loggedOut=True', () => {
+  beforeEach(() => {
```
Comment (kylemh) [inline cypress/integration/login.spec.js]: You can drop this `beforeEach` and `describe` and just put all of these tests in the top `describe`.

Comment (AllenAnthes) [reply cypress/integration/login.spec.js]: Good point.  When I started I didn't think I was going to use the same URL for each test.

**Pull request merged**